test_scripts/WebEngine/commonWebEngine.lua

```diff
@@ -121,7 +121,7 @@ function common.rejectedRegisterApp(pAppSessionId)
   :Do(function()
       local corId = session:SendRPC("RegisterAppInterface", common.getParams(pAppSessionId))
       common.getHMIConnection():ExpectNotification("BasicCommunication.OnAppRegistered"):Times(0)
-      session:ExpectResponse(corId, { success = false, resultCode = "REJECTED" })
+      session:ExpectResponse(corId, { success = false, resultCode = "DISALLOWED" })
```

Comment:
@YarikMamykin Please rename the function accordingly
(Make sure this action does not corrupt other scripts which use this function)

Reply:
@aderiabin fixed in https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/32/commits/b659b6e180dc8d1f24a80dfd02f05695b257ac04